test/en/en_time_exp.test.ts

```diff
@@ -287,3 +287,84 @@ test("Test - Parsing negative cases : 'at [some numbers] - [some numbers]' (Stri
 
     testUnexpectedResult(chrono.strict, "7-730");
 });
+
+test("Test - forward dates only option", function () {
+    testSingleCase(
+        chrono,
+        "  1am  ",
+        {
+            instant: new Date("Wed May 26 2022 01:57:00 GMT-0500 (CDT)"),
+            timezone: "CDT",
+        },
+        {
+            forwardDate: true
+        },
+        (result) => {
+            expect(result.index).toBe(2);
+            expect(result.text).toBe("1am");
+
+            expect(result.start).not.toBeNull();
+            expect(result.start.get("year")).toBe(2022);
+            expect(result.start.get("month")).toBe(5);
+            expect(result.start.get("day")).toBe(27);
+            expect(result.start.get("hour")).toBe(1);
+
+            expect(result.start).toBeDate(new Date(2022, 5 - 1, 27, 1));
```

Comment:
Nit: Recently, I think testing the date is redundant to testing date and hour about.